ElenaCzubiak-elena-cz/HW_4/BookManager/app/views/books/index.html.erb

```diff
@@ -2,6 +2,11 @@
 
 <div class="container">
 	<div class="margin-top-20">
+	<% if flash[:error] %>
+	    <div class="alert alert-danger">
+	        <%= flash[:error] %> 
+	   </div>    
+	   <% end %>
```

Comment:
Nice use of conditional flash helpers! Meaningful visual cues really do make a difference for users.